index.js

```diff
@@ -2,58 +2,98 @@
 
 const ANNOTATION_EXECUTOR_TYPE = 'beta.screwdriver.cd/executor'; // Key in annotations object that maps to an executor NPM module
 const Executor = require('screwdriver-executor-base');
+const Resque = require('node-resque');
+
+/**
+ * Strip the leading 'screwdriver-executor' from plugin names
+ * @method getPluginName
+ * @param {String} plugin
+ * @return {String}
+ */
+function getPluginName(plugin) {
+    return plugin.replace('screwdriver-executor-', '');
+}
 
-class ExecutorRouter extends Executor {
+class ExecutorQueue extends Executor {
     /**
      * Constructs a router for different Executor strategies.
      * @method constructor
      * @param  {Object}         config                      Object with executor and ecosystem
      * @param  {String}         [config.defaultPlugin]      Optional default executor
      * @param  {Object}         [config.ecosystem]          Optional object with ecosystem values
```

Comment:
Because the router takes in either a single object (which is when you provide the `config.ecosystem`) or an array of objects (which is when you *don't* provide the `config.ecosystem` and provide that value elsewhere)